build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -19,160 +19,115 @@ public static class PublishTargets
-                 { "Windows_x86", false },
-                 { "Windows_x64", false },
-                 { "Ubuntu_x64", false },
-                 { "RHEL_x64", false },
-                 { "OSX_x64", false },
-                 { "Debian_x64", false },
-                 { "CentOS_x64", false }
+                 { "build-Windows_x86", false },
+                 { "build-Windows_x64", false },
+                 { "build-Ubuntu_x64", false },
+                 { "build-RHEL_x64", false },
+                 { "build-OSX_x64", false },
+                 { "build-Debian_x64", false },
+                 { "build-CentOS_x64", false }
              };
 
-            List<string> blobs = new List<string>(AzurePublisherTool.ListBlobs($"{Channel}/Binaries/{CliNuGetVersion}/"));
-
-            var versionBadgeName = $"{CurrentPlatform.Current}_{CurrentArchitecture.Current}";
-            if (badges.ContainsKey(versionBadgeName) == false)
-            {
-                throw new ArgumentException("A new OS build was added without adding the moniker to the {nameof(badges)} lookup");
-            }
-
-            foreach (string file in blobs)
-            {
-                string name = Path.GetFileName(file);
-                string key = string.Empty;
-
-                foreach (string img in badges.Keys)
-                {
-                    if ((name.StartsWith($"{img}")) && (name.EndsWith(".svg")))
-                    {
-                        key = img;
-                        break;
-                    }
-                }
-
-                if (string.IsNullOrEmpty(key) == false)
-                {
-                    badges[key] = true;
-                }
-            }
+             IEnumerable<string> blobs = AzurePublisherTool.ListBlobs(AzurePublisher.Products.SDK, CliVersion);
+             foreach (string blob in blobs)
+             {
+                 if (blob.StartsWith("build"))
+                 {
+                     string name = blob.Substring(0, blob.IndexOf("."));
+                     if (monikers.ContainsKey(name))
+                     {
+                         monikers[name] = true;
+                     }
+                     else
+                     {
+                         throw new ArgumentOutOfRangeException("Unknown OS moniker found during build finalization");
+                     }
+                 }
+             }
+
+             return monikers.Keys.All(k => monikers[k]);
+        }
+        
+        public static void PublishCliVersionInformation(BuildTargetContext c)
+        {
+            var versionBadge = c.BuildContext.Get<string>("VersionBadge");
+            VersionUpdater.PublishVersionBadge(versionBadge, Channel, Pointer).Wait();
+            VersionUpdater.PublishVersionFile("build.version", Utils.GetCliVersionFileContent(c), Channel, Pointer).Wait();
+        }
 
-            return badges.Keys.All(key => badges[key]);
+        public static void PublishCliNupkgs(BuildTargetContext c)
```

Comment:
This isn't publishing the packages but rather just updating the `dotnet/versions` repo, shouldnt the name reflect that?